src/components/dashboard/TeamMembersTable/TeamEditInfo.js

```diff
@@ -21,32 +21,32 @@ import {
 const EditTeamInfo = ({ setOnEditPage, teamId, teamName, teamOrg }) => {
   const toast = useToast();
   const dispatch = useDispatch();
-  const [newTeamName, setNewTeamName] = useState('');
-  const [newTeamOrg, setNewTeamOrg] = useState('');
-  const [editTeamName, { data: dataNameChange, loading: loadingNameChange }] = useMutation(UPDATE_TEAM_NAME);
-  const [editTeamOrg, { data: dataOrgChange, loading: loadingOrgChange }] = useMutation(UPDATE_TEAM_ORGANIZATION);
-
-  useEffect(() => {
-    if (dataNameChange) {
-      const name = dataNameChange.updateTeamName.name;
-      dispatch({ type: UPDATE_USER_TEAM_NAME, payload: name });
-      toast({
-        position: 'top',
-        render: props => <Toast {...props} description="You have updated your team name" isClosable />,
-      });
-    }
-  }, [dataNameChange, dispatch, toast]);
+  const [newTeamName, setNewTeamName] = useState(teamName);
+  const [newTeamOrg, setNewTeamOrg] = useState(teamOrg);
+  const [editTeam, { data: dataTeamEdit, loading: loadingTeamEdit, error: errorTeamEdit }] = useMutation(
+    UPDATE_TEAM_NAME_ORG,
+  );
 
   useEffect(() => {
-    if (dataOrgChange) {
-      const organization = dataOrgChange.updateTeamOrganization.organization;
-      dispatch({ type: UPDATE_USER_TEAM_ORGANIZATION, payload: organization });
-      toast({
-        position: 'top',
-        render: props => <Toast {...props} description="You have updated your team affiliation" isClosable />,
-      });
+    if (dataTeamEdit) {
+      const organization = dataTeamEdit.updateTeamNameOrg.organization;
+      const name = dataTeamEdit.updateTeamNameOrg.name;
+      if (organization !== teamOrg) {
+        dispatch({ type: UPDATE_USER_TEAM_ORGANIZATION, payload: organization });
+        toast({
+          position: 'top',
+          render: props => <Toast {...props} description="You have updated your team affiliation" isClosable />,
+        });
+      }
+      if (name !== teamName) {
+        dispatch({ type: UPDATE_USER_TEAM_NAME, payload: name });
+        toast({
+          position: 'top',
+          render: props => <Toast {...props} description="You have updated your team name" isClosable />,
+        });
```

Comment:
we could combine this into one toast nicely with template literal strings — although idk if it'll look nicer or worse

```jsx
const updatedName = ...
const updatedOrg = ...
const updatedBoth = ...

description={`You have updated ${updatedTeam ? "your team" : ""} ${updatedBoth ? "and " : ""} ${updatedOrg ? "your org" : ""}`}
```

Reply:
I chose my method so it's easier to tell how many things changed at a glance rather than having to read the message

Reply:
sg, let's keep it that way